app.py

```
@@ -2,50 +2,199 @@
+            <button class="accessibility-button" onclick="document.documentElement.style.setProperty('--font-scale', '1')">A</button>
+            <button class="accessibility-button" onclick="document.documentElement.style.setProperty('--font-scale', '0.5')">A-</button>
+        </div>
+    """
+    )
+
+    # ==============================================================
+    # Hauptinterface
+    # ==============================================================
 
-ls_ui = gr.Interface(
     simplify_text,
```

Ruff (0.8.2)

Found useless expression. Either assign it to a variable or remove it.

(B018)